src/components/members.component.js

```diff
@@ -60,8 +60,13 @@ const styles = StyleSheet.create({
     paddingLeft: 15,
   },
   flatList: {
-    marginLeft: 15,
-    marginRight: 15,
+    paddingLeft: 15,
+    paddingRight: 15,
```

Comment:
This can be simplified: instead of `paddingLeft` and `paddingRight`, use `paddingHorizontal`.